pkg/api/validation/v1beta1-swagger.json

```diff
@@ -1312,6 +1312,31 @@
      }
     }
    },
+   "v1beta1.ISCSI": {
```

Comment:
is this all that update-swagger-spec.sh generated?  I would expect v1beta2 and v1beta3, too.